app/seed-data/formula_values/scripts/EFDB_IPCC_Waste.py

```diff
@@ -0,0 +1,624 @@
+    seed_string = get_filename()
+    seed_value = string_to_hash(seed_string)
+    set_seed(seed_value)
+
+    # output directory
+    output_dir = "../data_processed/EFDB_2006_IPCC_guidelines/"
+    output_dir = os.path.abspath(output_dir)
+    make_dir(path=Path(output_dir).as_posix())
+
+    # raw data file path
+    input_fl = "../data_raw/EFDB_2006_IPCC_guidelines/EFDB_output.xlsx"
+    input_fl = os.path.abspath(input_fl)
+
+    # =================================================================
+    # Publisher
+    # =================================================================
+    publisher_data = {
+        "name": "IPCC",
+        "URL": "https://www.ipcc.ch/",
+    }
+    publisher_data["publisher_id"] = uuid_generate_v3(name=publisher_data.get("name"))
+
+    write_dic_to_csv(output_dir, "Publisher", publisher_data)
+
+    # =================================================================
+    # DataSource
+    # =================================================================
+    datasource_data = {
+        "datasource_name": "IPCC",
+        "dataset_name": "IPCC Emission Factor Database (EFDB) [2006 IPCC Guidelines]",
+        "URL": "https://www.ipcc-nggip.iges.or.jp/EFDB/main.php",
+        "publisher_id": publisher_data.get("publisher_id"),
+    }
+    datasource_data["datasource_id"] = uuid_generate_v3(
+        name=datasource_data.get("dataset_name")
+    )
+
+    write_dic_to_csv(output_dir, "DataSource", datasource_data)
+
+    # =================================================================
+    # Formula Input Values
+    # =================================================================
+    conn = duckdb.connect(':memory:')
+
+    query = f"""
+            INSTALL spatial;
+            LOAD spatial;
+            
+            CREATE OR REPLACE TABLE waste_ef AS 
+            SELECT "EF ID" AS ef_id,
+            REPLACE(REGEXP_REPLACE("IPCC 1996 Source/Sink Category", '\n', '|'), CHR(10), '|') AS ipcc_sector_multi,
+            REPLACE(REGEXP_REPLACE("Gas", '\n', '|'), CHR(10), '|')  AS gas_multi,
+            "Fuel 1996" AS fuel_1996,
+            "Fuel 2006" AS fuel_2006,
+            "C pool" c_pool,
+            "Type of parameter" AS type_parameter,
+            Description,
+            "Technologies / Practices" AS technologies_paractises,
+            "Parameters / Conditions" AS parameters_conditions,
+            "Region / Regional Conditions" AS region,
+            "Abatement / Control Technologies" AS control_paractises,
+            "Other properties" AS properties,
+            Value AS emissionsfactor_value,
+            Unit AS emissionsfactor_units,
+            Equation AS ipcc_equation,
+            "IPCC Worksheet" as ipcc_worksheet,
+            "Technical Reference" as technical_reference,
+            "Source of data" as dataset_name,
+            "Data provider" AS data_source
+            FROM ST_READ('/Users/amandaeames/Downloads/EFDB_output.xlsx')
```

Comment:
Replace hardcoded path with either a relative path or at least a constant on the top. This will not work on other machines.